marklogic-data-hub/src/main/resources/ml-modules/services/mlCollections.sjs

```diff
@@ -31,10 +32,19 @@ function get(context, params) {
       docs = docs.toArray();
       let i;
       for (i in docs) {
-        let obj = {
-          "uri": xdmp.nodeUri(docs[i]),
-          "docs": docs[i]
-        };
+        let obj;
+        if(urisOnly === 'true') {
```

Comment:
if(urisOnly) {

Reply:
From SearchService.Collections , 'urisOnly' is passed as a string. So, if urisOnly is set to "false" (or any other string),if(urisOnly) will be true, hence this check if(urisOnly === 'true')